examples/ItemBack/ItemBack.ino

```diff
@@ -9,22 +10,23 @@
 
 extern MenuItem* settingsMenu[];
 
+// Create submenu and precise its parent
+// clang-format off
+MENU_SCREEN(settingsScreen, settingsItems,
+    ITEM_BASIC("Backlight"),
+    ITEM_BASIC("Contrast"),
+    ITEM_BACK());
+// clang-format on
+
 // Define the main menu
-MAIN_MENU(
+// clang-format off
+MENU_SCREEN(mainScreen, mainItems,
     ITEM_BASIC("Start service"),
     ITEM_BASIC("Connect to WiFi"),
-    ITEM_SUBMENU("Settings", settingsMenu),
+    ITEM_SUBMENU("Settings", settingsScreen),
     ITEM_BASIC("Blink SOS"),
     ITEM_BASIC("Blink random"));
```

Comment:
Same goes here, please find a way to make them properly ordered.
That's the reason I opted to have the externs defined outside, so that you can order the menu's in a way that it's not visually confusing to understand.